loopy/target/pyopencl_execution.py

```diff
@@ -355,6 +355,7 @@ def __call__(self, queue, *,
 
         assert entrypoint is not None
 
+        self.check_for_required_arguments(kwargs.keys())
```

Comment:
* C execution should probably have this, too?
```suggestion
        if __debug__:
            self.check_for_required_arguments(kwargs.keys())
```

Reply:
Both done: 279a86f